src/lib/utils/file-manager.ts

```diff
@@ -117,7 +117,8 @@ export const calculateUrlsWithMetadataToChange = (
                                     mediaType: MediaType.Text,
                                     contentId: resourceMenuItem.contentId,
                                     url: resourceContentApiFullUrl(resourceMenuItem),
-                                    size: resourceMenuItem.contentSize,
+                                    size: resourceMenuItem.contentSize + (resourceMenuItem.inlineMediaSize ?? 0),
+                                    hasInlineMedia: resourceMenuItem.inlineMediaSize !== null,
```

Comment:
Do we want these to be dependent on what the user has selected in terms of text or audio or media? It seems like if they only have `text` selected on the bottom bar then we wouldn't want to pull media.